package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "hast-util-to-html",
-  "version": "8.0.3",
-  "description": "hast utility to serialize to HTML",
+  "name": "hast-util-to-html-ns",
```

Comment:
I don't think renaming the package is an intentional part of this change?

Reply:
This was an accident. I'll delete this one and send another PR up.

Reply:
okay, it doesn't strictly need to be another PR.
It's also possible to walk back a commit from a PR

```bash
# revert one commit
git reset --hard HEAD~1
# push changes to remote git repository
git push origin override-entities-subset --force-with-lease
```

Reply:
I just saw your comment. Is there a benefit to this over a git revert?

Reply:
The difference is:
`reset --hard` can be used to remove piece(s) of history.
`revert` adds more history, with the exact opposite changes.

It depends on the intended goal, if the change wasn't intended in the first place, adding two pieces for history may not make sense.

Reply:
Okay, thank you for clarifying that. I'll keep that in mind for the next time.